src/api.ts

```diff
@@ -330,16 +334,34 @@ function createApi({
         'INSERT INTO Tileset (id, tilejson, format, upstreamTileUrls) ' +
           'VALUES (:id, :tilejson, :format, :upstreamTileUrls)'
       ).run({
-        id,
+        id: tilesetId,
         format: tilejson.format,
         tilejson: JSON.stringify(tilejson),
         upstreamTileUrls: JSON.stringify(tilejson.tiles),
       })
 
       const result = {
         ...tilejson,
-        tiles: [getTileUrl(id)],
-        id,
+        id: tilesetId,
+        tiles: [getTileUrl(tilesetId)],
+      }
+
+      // Create raster style if tileset format is a raster format
+      if (tilejson.format !== 'pbf') {
```

Comment:
possible adjustment: be more explicit about which formats we should create a style for i.e. `['jpg', 'png', 'webp'].includes(tilejson.format)`